iset.mm

```diff
@@ -63863,6 +63893,54 @@ all reals (greatest real number) for which all positive reals are greater.
       KVBFVKVDTVEWHXFAWKCXDWGXEDWCWKPVFRVHTVLVM $.
   $}
 
+  ${
+    $d A x y $.  $d B x y $.
+    $( A disjoint union is a subset of a Cartesian product.  (Contributed by
+       AV, 25-Jun-2022.) $)
+    djuss $p |- ( A |_| B ) C_ ( { (/) , 1o } X. ( A u. B ) ) $=
+      ( vx vy c0 c1o cv wcel cinl cfv wceq wrex cinr wa cop cvv cmpt a1i adantr
+      simpr cdju cpr cun cxp wo djur df-inl weq adantl elex 0ex vex opex fvmptd
+      opeq2 eqtrd elun1 prid1 jctil opelxp sylibr eqeltrd rexlimiva df-inr con0
+      1on elexi elun2 prid2 jaoi syl ssriv ) CABUAZEFUBZABUCZUDZCGZVMHVQDGZIJZK
+      ZDALZVQVRMJZKZDBLZUEVQVPHZDABVQUFWAWEWDVTWEDAVRAHZVTNZVQEVROZVPWGVQVSWHWF
+      VTTWFVSWHKVTWFCVREVQOZWHPIPICPWIQKWFCUGRCDUHZWIWHKWFVQVREUOUIVRAUJWHPHWFE
+      VRUKDULZUMRUNSUPWGEVNHZVRVOHZNZWHVPHWFWNVTWFWMWLVRABUQEFUKURUSSEVRVNVOUTV
+      AVBVCWCWEDBVRBHZWCNZVQFVROZVPWPVQWBWQWOWCTWOWBWQKWCWOCVRFVQOZWQPMPMCPWRQK
+      WOCVDRWJWRWQKWOVQVRFUOUIVRBUJWQPHWOFVRFVEVFVGZWKUMRUNSUPWPFVNHZWMNWQVPHWP
+      WMWTWOWMWCVRBAVHSEFWSVIUSFVRVNVOUTVAVBVCVJVKVL $.
+  $}
+
+  $( The first component of an element of a disjoint union is either ` (/) ` or
+     ` 1o ` .  (Contributed by AV, 26-Jun-2022.) $)
+  eldju1st $p |- ( X e. ( A |_| B )
+                   -> ( ( 1st ` X ) = (/) \/ ( 1st ` X ) = 1o ) ) $=
+    ( cdju c0 c1o cpr cun cxp wss wcel c1st cfv wceq wo djuss ssel2 xp1st elpri
```

Comment:
Using djur or djuun doesn't make the proof shorter ? Or conversely does eldju1st (and eldju2ndl/r) allow shortenings of these ?

Reply:
I tried to prove ~eldju1st with ~djur, but the proof becomes longer. I do not expect many gains playing around with the other combinations, therefore I would not work on this further.